pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -425,6 +427,16 @@ func (r *ReconcileVirtualMachineImport) createVM(provider provider.Provider, ins
 		return "", err
 	}
 
+	if instance.Spec.VMDescription != nil {
+		desc := utils.MakeLabelFrom(*instance.Spec.VMDescription)
```

Comment:
Shortening is not needed here unless there is a business requirement - the limit of 63 characters is imposed on the annotation name and on labels.
(reference: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/).

Reply:
Thanks, will remove.